core/ui/ViewTemplate/import.tid

```diff
@@ -3,8 +3,17 @@ tags: $:/tags/ViewTemplate
 
 \define lingo-base() $:/language/Import/
 
+\define confirmCancel()
+<$wikify name=message text={{$:/language/Import/Listing/Cancel/Warning}}>
```

Comment:
I think it would be preferable in the core to always quote attribute values: `name="message"`

Reply:
changed it in the next commit